### DISCUSSION TITLE
Add Malayalam Channels

### DISCUSSION THREAD

**Discussion by friend8951** [iptv-org/community#429] [Q&A] [answered]
Hi
Can you please add the below Malayalam channels to Malayalam m3u playlist. These channels are already there in the India all language playlist.
KITE VICTERS KERALA
Mandra TV
Metro 7
Next TV Malabar
NVision
Pulari Cinema
Thanks

**Comment by Anoopplavilayil123** [comment]
Flowers

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel
